ledger/db/src/test_utils/mock_ledger.rs

```diff
@@ -300,11 +301,139 @@ pub fn get_test_ledger_blocks(n_blocks: usize) -> Vec<(Block, BlockContents)> {
     blocks_and_contents
 }
 
+/// Get blocks with custom content in order to simulate conditions seen in
+/// production
+///
+/// * `outputs_per_recipient_per_block` - transaction outputs per account per
+///   block
+/// * `num_accounts` - number of accounts in the blocks
+/// * `num_blocks` - number of simulated blocks to create
+/// * `key_images_per_block` - number of key images per block
+/// * `max_token_id` - number of distinct token ids in blocks
+pub fn get_custom_test_ledger_blocks(
+    outputs_per_recipient_per_block: usize,
+    num_accounts: usize,
+    num_blocks: usize,
+    key_images_per_block: usize,
+    max_token_id: u64,
+) -> Vec<(Block, BlockContents)> {
+    let mut rng: StdRng = SeedableRng::from_seed([1u8; 32]);
+
+    // Number of total tx outputs in all blocks
+    let num_outputs: u64 =
+        (num_accounts * outputs_per_recipient_per_block * num_blocks * (max_token_id as usize + 1))
+            as u64;
+    assert!(num_outputs >= 16);
```

Comment:
why do there need to be 16 outputs?  Should this limitation be communicated in the function doc comment, or is this a general block thing that I'm just not aware of?

Reply:
It's a chain constant, it's in the method above it. I'd rather not change the entire test file.

Reply:
> Yeah that's a fixed constant in the mobilecoin chain.

Where is this constant?

Reply:
// Where is this constant?

https://block-explorer.mobilecoin.foundation/block/0

It was the number of outputs in the genesis block is and more or less the universally used number when initializing test ledgers.